src/test/scala/io/citrine/lolo/learners/ExtraRandomTreesTest.scala

```diff
@@ -0,0 +1,177 @@
+package io.citrine.lolo.learners
+
+import breeze.stats.distributions.Beta
+import io.citrine.lolo.TestUtils
+import io.citrine.lolo.stats.functions.Friedman
+import org.junit.Test
+
+import scala.util.Random
+
+@Test
+class ExtraRandomTreesTest {
+  val rng = new Random(92345L)
+
+  /**
+    * Test that the regression does the same thing as the regression bagger
+    */
+  @Test
+  def testRegression(): Unit = {
+    rng.setSeed(982534L)
+    val trainingData = TestUtils.binTrainingData(
+      TestUtils.generateTrainingData(1024, 12, noise = 0.1, function = Friedman.friedmanSilverman, seed = rng.nextLong()),
+      inputBins = Seq((0, 8))
+    )
+
+    Seq(true, false).foreach { randomlyRotateFeatures =>
+      val RFMeta = ExtraRandomTrees(randomlyRotateFeatures = randomlyRotateFeatures, rng = rng)
+        .train(trainingData)
+      val RF = RFMeta.getModel()
+
+      assert(RFMeta.getLoss().get < 1.0, "Loss of bagger is larger than expected")
+
+      val results = RF.transform(trainingData.map(_._1))
+      // val means = results.getExpected()
+      val sigma: Seq[Double] = results.getUncertainty().get.asInstanceOf[Seq[Double]]
+      assert(sigma.forall(_ >= 0.0))
```

Comment:
This vector round-off level small for me: values are < 1e-12.  If bootstrapping is turned off, I'm pretty sure the jackknife method will fail or at least do something funny.